src/git.rs

```diff
@@ -201,6 +203,8 @@ impl Repository {
         self.repository
             .commit(Some("HEAD"), &sig, &sig, &msg, &tree, &[&parent])?;
 
+        println!("commit_and_push: Pushing \"{}\"", msg);
```

Comment:
I think we can drop this output, and have just 2 log entries generated per operation.  One at the beginning and a success or failure at the end.